src/main/java/com/terraformersmc/cinderscapes/init/CinderscapesFeatures.java

```diff
@@ -148,11 +152,11 @@ public static void init() {
 
         DEAD_TREE = Registry.register(Registry.FEATURE, Cinderscapes.id("dead_tree"), new DeadTreeFeature());
 
-        UniformIntDistribution spread = UniformIntDistribution.of(3, 4); // 3-7
-        SOUL_SAND_REPLACE_CONFIG = new NetherrackReplaceBlobsFeatureConfig(Blocks.NETHERRACK.getDefaultState(), Blocks.SOUL_SAND.getDefaultState(), spread);
-        SOUL_SOIL_REPLACE_CONFIG = new NetherrackReplaceBlobsFeatureConfig(Blocks.NETHERRACK.getDefaultState(), Blocks.SOUL_SOIL.getDefaultState(), spread);
-        GRAVEL_REPLACE_CONFIG = new NetherrackReplaceBlobsFeatureConfig(Blocks.NETHERRACK.getDefaultState(), Blocks.GRAVEL.getDefaultState(), spread);
-        CRIMSON_NYLIUM_REPLACE_CONFIG = new NetherrackReplaceBlobsFeatureConfig(Blocks.NETHERRACK.getDefaultState(), Blocks.CRIMSON_NYLIUM.getDefaultState(), spread);
-        BLACKSTONE_REPLACE_CONFIG = new NetherrackReplaceBlobsFeatureConfig(Blocks.NETHERRACK.getDefaultState(), Blocks.BLACKSTONE.getDefaultState(), spread);
+        IntProvider spread = UniformIntProvider.create(3, 4); // 3-7
```

Comment:
`UniformIntDistribution` uses `min` and `height`, while `UniformIntProvider` uses `min` and `max`:

```suggestion
        IntProvider spread = UniformIntProvider.create(3, 7);
```